src/app/meetups/meetups.component.ts

```diff
@@ -4,20 +4,34 @@ import { MatPaginator, MatTableDataSource, MatSort, MatDialog } from '@angular/m
 import { DialogsPromptComponent } from '../shared/dialogs/dialogs-prompt.component';
 import { PlanetMessageService } from '../shared/planet-message.service';
 import { filterSpecificFields } from '../shared/table-helpers';
+import { SelectionModel } from '@angular/cdk/collections';
+import { Router } from '@angular/router';
 
 @Component({
   templateUrl: './meetups.component.html',
+  styles: [ `

```

Comment:
this style has already been moved to some shared css file so it might not be required to add it here

Reply:
It was supposed to change since it has been removed from master. Not all master changes reflect in this branch. Don't know why

Reply:
@lmmrssa I don't think this made it to a shared file yet.  I made a separate issue #458 about reducing repetition with the implementations of this layout.